### DISCUSSION TITLE
Supabase REST API cannot match with UUID?

### DISCUSSION THREAD

**Discussion by imknight** [supabase/supabase#3369] [Questions] [answered]
I m trying to query the API with profile_id=eq.<uuid> through REST API, but getting invalid input syntax for type uuid

**Comment by steve-chavez** [comment, marked as answer]
> I m trying to query the API with profile_id=eq. through REST API, but getting invalid input syntax for type uuid

Share the sample `uuid` you're trying to specify. Probably it's invalid as the error mentions.

**Reply by imknight** [reply]
thanks for the reminder, seems like a not complete UUID causing that.
